code/modules/mob/living/basic/trooper/russian.dm

```diff
@@ -26,14 +26,16 @@
 	r_hand = /obj/item/gun/ballistic/automatic/pistol
 	loot = list(
 		/obj/effect/mob_spawn/corpse/human/russian/ranged,
-		/obj/item/gun/ballistic/revolver/nagant,
+		/obj/item/gun/ballistic/revolver/sakhnomanni,
 	)
-	var/casingtype = /obj/item/ammo_casing/n762
+	var/casingtype = /obj/item/ammo_casing/short310
 	var/projectilesound = 'sound/weapons/gun/revolver/shot.ogg'
+	var/burstshots = 2
+	var/burstintervals = 1.5
```

Comment:
There's not honestly a huge number of reasons to make these into vars unless you expect this mob to have subtypes which modify these values.